exercises/Python_Principles_4.ipynb

```diff
@@ -720,18 +650,36 @@
    "metadata": {},
    "source": [
     "\n",
-    "Here the numbers `1`, `2` and `3` are printed. We assign `1` to the variable `i`. The while loop evaluates `i < 6`, print the numbers and then assigns one above the current value of `i`. In the case that `i` is 3, the integer `3` is printed. Then the if statement will be run and we break the while loop. Notice if we hadn't included the break statement i would keep being reasigned a new number.\n",
+    "Here the numbers `1`, `2` and `3` are printed. We assign `1` to the variable `i`. The while loop evaluates `i < 6`, print the numbers and then assigns one above the current value of `i`. In the case that `i` is 3, the integer `3` is printed. Then the if statement will be run and we break the while loop. \n",
     "\n",
     "When you don't know how many values or number of items are in a data type use a while loop."
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Write a code to run the while loop forever but break it if `i` > 1000"
```

Comment:
@AlexTeslenko Similarly to last comment, write some code instead ? of write a code?

Reply:
fixed all such cases